docs/faq.md

```diff
@@ -111,6 +111,16 @@ Also Shorebird does not support changing native code (e.g. Java/Kotlin on
 Android or Objective-C/Swift on iOS). The tool will warn you during an
 attempted patch if you have changed native code.
 
+### Can I use Shorebird for all my Dart changes?
+
+Technically speaking, yes, a Shorebird patch can update any Dart code.
```

Comment:
```suggestion
Shorebird can be used to update any and all Dart code in your app, including non-plugin dependencies.
```

Reply:
Maybe we should say "include dart packages (not plugins)."

Reply:
"including packages that do not contain plugins"?

Reply:
I went with "including pure dart packages" lmk what you think